.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
     "typescriptreact"
   ],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
why do we need this? wont this disable linting on save?

Reply:
it is supposed to have one value  "source.fixAll.eslint": true 

Do I fix and recreate a pull request?

Reply:
TO disbale linting on save, I would have to set 

`  "source.fixAll.eslint": false `
  which is not the case here

Reply:
but we want to lint on save,
did it cause any issue when you did that? just trying to understand if it had caused you any problem for you to push that change.

Reply:
Oh no it didn't. Only thing that happened was I upgraded my VS code to a newer version, probably that is where it was added from